praw/models/reddit/subreddit.py

```diff
@@ -480,7 +480,7 @@ def subscribe(self, other_subreddits=None):
             list of subreddits.
 
         """
-        data = {'action': 'sub', 'skip_inital_defaults': True,
```

Comment:
I've just tested and it does change behavior. Default is False. 

If someone creates an account and the very first subscription is made through PRAW, before the account would also be subscribed to all the default subs, while the "fix" avoids adding the defaults.

I'll revert this one, not a typo fix. 

Maybe the right fix could be to keep the current, arguably sane, default behavior (i.e., set `skip_initial_defaults` to `False`) and make it possible to pass the option when calling subscribe (with documentation stressing it only has effects on the very first subscription). Whatever you figure would be a good way forward, I can try to get it done sometime soon.